tests/test_hdf_ioc.py

```diff
@@ -34,38 +34,10 @@
 
 from pandablocks_ioc._hdf_ioc import HDF5RecordController
 
-NAMESPACE_PREFIX = "HDF-RECORD-PREFIX"
+NAMESPACE_PREFIX = "HDF-RECORD-PREFIX-" + str(uuid4())[:4].upper()
 HDF5_PREFIX = NAMESPACE_PREFIX + ":HDF5"
 
 
-def chunked_read(f: BufferedReader, size: int) -> Iterator[bytes]:
-    data = f.read(size)
-    while data:
-        yield data
-        data = f.read(size)
-
-
-@pytest_asyncio.fixture
-def slow_dump():
-    with open(Path(__file__).parent / "slow_dump.txt", "rb") as f:
-        # Simulate small chunked read, sized so we hit the middle of a "BIN " marker
-        yield chunked_read(f, 44)
-
-
-@pytest_asyncio.fixture
-def fast_dump():
-    with open(Path(__file__).parent / "fast_dump.txt", "rb") as f:
-        # Simulate larger chunked read
-        yield chunked_read(f, 500)
-
-
-@pytest_asyncio.fixture
-def raw_dump():
-    with open(Path(__file__).parent / "raw_dump.txt", "rb") as f:
-        # Simulate largest chunked read
-        yield chunked_read(f, 200000)
-
-
```

Comment:
Are there some txt files that can be deleted from the repo now?

Reply:
Yep seems like fast and slow dump aren't really used as files, neither are the `x_dump` fixtures. I'll take them all out